packages/core/src/Stores/Helpers/chart-markers.js

```diff
@@ -68,13 +68,14 @@ const addLabelAlignment = (tick, idx, arr) => {
 
 const createTickMarkers = contract_info => {
     const is_accumulator = isAccumulatorContract(contract_info.contract_type);
-    const is_contract_closed = contract_info.status && contract_info.exit_tick_time;
+    const is_contract_closed = contract_info.status !== 'open';
```

Comment:
I think if `contract_info.status` isn't open, it doesn't mean that it's closed for all markets. There are other statuses available like `profit` or `loss` which are assigned while contract is open. Was it considered in this logic?

Reply:
@akmal-deriv contract_info.status is always 'open' when a contract is ongoing for all trade types. Other contract statuses include: lost, won, and sold (for Multipliers). In other words, if the contract status is not 'open', it's no longer ongoing which is what I need to check here.🙂